apache-pulsar/src/main/java/io/debezium/examples/apache/pulsar/config/PropertyLoader.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.examples.apache.pulsar.config;
+
+import org.apache.log4j.Logger;
+
+import java.util.Enumeration;
+import java.util.Properties;
+
+
+public class PropertyLoader {
+    private static Logger logger = Logger.getLogger(PropertyLoader.class);
```

Comment:
Do not mix slf4j and plain log4j logging in a single project

Reply:
Done.